### ISSUE TITLE
Allow to slice even thoght colisions may ocur

### ISSUE THREAD

**Issue by alc112** [SoftFever/OrcaSlicer#6601]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

I wouldn't say it's a problem. The way this is set now is safe for beginners.

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

When I want to print something in "Print Sequence by Object" mode and Orca detects a potential collision, it simply won't let me slice the project even if I set Extruder Clearance radius to zero. 

### Describe alternatives you've considered

If I'm fully aware of the risk and I know that nothing will happen, I'd like a way to bypass this limitation. Perhaps a pop-up warning where Orca explains the potential risks and allows me to proceed if I choose to.

### Additional context

I'm making keychains of 2 layers thick. This thickness is not enough to cause collisions. In fact, I do this all the time in other slicers but Orca just won't let me continue.

**Comment by bo0tzz** [comment]
I don't think this is stale

**Comment by MAJCcustoms** [comment]
I'm not sure why there isn't an option to override it or disable collision detection altogether. I've already tried lowering the extruder clearance to 0 and it still doesn't let me slice

**Comment by loutwice** [comment]
@alc112  i think we got the same idea ,i just made my ticket https://github.com/SoftFever/OrcaSlicer/issues/8608

**Comment by dannypike** [comment]
I would like to have this option, too. I have a project where Orca thinks that two objects 70mm apart are colliding, but the same pattern will not collide when they are 40mm apart. Yet, neither is true

It may be something to do with the order in which it is scanning, but it shouldn't be. Orca Slicer needs to allow us to decide the order in which objects are printed, and also to allow us to override its idea of whether they will collide at all, please!

Beginners will be using the manufacturer's program (Bambu etc). If you've got as far as using a different slicer, then you should have enough knowledge and skill to be allowed to override "safety" settings like this one. The worse that will happen is a loud noise and some wasted material.

**Comment by MoreSly** [comment]
I would also like an override in advanced mode.

**Comment by SoongJr** [comment]
Definitely. For me it's that you can only give a radius for extruder clearance, but obviously the actual clearance is not circular.
Ideally we could set two radii, one for warning and one for error. That would make it easier to approximate.
Even better yet, an STL model of extruder, vents and stuff, but that's overkill 🤣 

A simple "yes I see what you mean, and I have ensured that there is no collision in that direction" would be sufficient.

At least for me it does work to set extruder clearance to 0 with orca 2.3.1, so I just need to ensure to set it back before changing any positions...

**Comment by EnduringGuerila** [comment]
I would also really like an option to allow override for some prints. 

Example, these are so tiny, they'll lever collide with my print head:
<img width="1267" height="733" alt="Image" src="https://github.com/user-attachments/assets/0e7cdd4f-1ba2-4d94-ae32-ec1408887111" />

I really like print by object since if I have a failure, it's way less likely to ruin the other parts. 
Sure, this print I could just print by layer for the bottom pieces and then print the box after the 7 connectors finish... but still not as ideal.

**Comment by dannypike** [comment]
The problem is that the code wants to prevent any exclusion zone from overlapping another exclusion zone.

My printer has only one nozzle, so it will never encounter another toolhead. So Orca Slicer only needs to ensure that it's exclusion zone does not overlap another object; not another exclusion zone.

This subtle distinction makes a very big difference when you have lots of small objects that you're trying to cram onto a single sheet.

**Comment by donbarthel** [comment]
SuperSlicer allows this but I'd rather use OrcaSlicer. I *know* that there won't be a collision because I've made a multi-layer cutout in one color/part to accommodate another color/part. And I'm careful to z-hop over previously printed parts.

**Comment by dkowis** [comment]
A use in my case is that I've got exclusion zones defined for the filament purge, but if I'm only using a single filament, I don't need to worry about the purge zone, and so it'd be super nice for me to say "nope, don't worry about the exclusion zone, cause I'm not changing filaments."
